docs/book/src/developers/development.md

```diff
@@ -19,7 +19,7 @@
 5. Install [Kustomize][kustomize]
    - `brew install kustomize` on macOS.
    - [install instructions](https://kubectl.docs.kubernetes.io/installation/kustomize/) on Windows + WSL2.
-   - [install instructions][kustomizelinux] on Linux
+   - [install instructions](https://kubectl.docs.kubernetes.io/installation/kustomize/) on Linux
```

Comment:
Should we just conflate this into one line given that the link is now the same?

Should we also drop the now unused kustomizelinux link at the bottom of the doc?

Reply:
Sounds good to me. Will do.